### ISSUE TITLE
Focusing an empty Space sometimes also focuses my other display

### ISSUE THREAD

**Issue by 7vq** [koekeishiya/yabai#1569]
1. Have windows from one application visible on both displays, and one of that application's windows is currently focused
2. `yabai -m space --focus <prev|next>` to an empty Space immediately adjacent to one of the initially active Spaces
3.  yabai focuses the empty Space as expected, but also instantly focuses the app's other window on the other display

For my setup right now at least, this **only occurs if the initial display** (that has both the initially focused Space and the empty Space I'm switching to) **is my secondary monitor**.

Sounds like this has been a hard-to-pin-down bug since at least March: https://github.com/koekeishiya/yabai/issues/1033#issuecomment-1079985273, but I did see this bug even when using yabai's sample config

I saw this earlier too like that other post, but FWIW I'm using macOS 13.0.1; yabai stable 5.0.1, HEAD; HEAD-03b2483

**Comment by Bellavene** [comment]
This is not a bug, turn off "Displays have separate spaces" in System Settings.

**Comment by 24unix** [comment]
> This is not a bug, turn off "Displays have separate spaces" in System Settings.

I though yabai would even start if that option is disabled. I'd be more then happy if yabai would run in that setup.

**Comment by Bellavene** [comment]
Sorry then

**Comment by 7vq** [comment]
> I though yabai would even start if that option is disabled

Yeah, that's what the wiki has always said at least.

&nbsp;

@24unix if this bug affects you too, it'd be great if you could thumbs up the OP, because [issues can be sorted by the number of thumbs up the OP gets](https://github.com/koekeishiya/yabai/issues?page=1&q=is%3Aissue+is%3Aopen+sort%3Areactions-%2B1-desc). @albert-ying if it's still affecting you as well a thumbs up would be great.

**Comment by 24unix** [comment]
> @24unix if this bug affects you too, it'd be great if you could thumbs up the OP, 


No, I'm affected by the fact that we have to break spaces to use yabai.

**Comment by 7vq** [comment]
> No, I'm affected by the fact that we have to break spaces to use yabai.

Oh. I don't really care about that myself, but you could thumbs up [that issue that you already commented on](https://github.com/koekeishiya/yabai/issues/963) instead. Though looks like that's not going to change.

**Comment by 24unix** [comment]
> > No, I'm affected by the fact that we have to break spaces to use yabai.
> 
> Oh. I don't really care about that myself, but you could thumbs up [that issue that you already commented on](https://github.com/koekeishiya/yabai/issues/963) instead. Though looks like that's not going to change.

Exactly, it was told more than once that yabai needs this setup. Which was no problem while TotalSpaces was functional.
